### ISSUE TITLE
Compile build with op support for int32

### ISSUE THREAD

**Issue by phildow** [doc-ai/tensorflow-ios-framework#5]
Get a number of related errors when trying to load graphs that don't cast custom layers to float32 before using them elsewhere:

```
 Non-OK-status: LoadSavedModel(session_opts, run_opts, model_dir, tags, &_saved_model_bundle) status: Invalid argument: No OpKernel was registered to support Op 'Maximum' used by {{node gradients/Mean_grad/Maximum}}with these attrs: [T=DT_INT32, _output_shapes=[[]]]
Registered devices: [CPU]
Registered kernels:
  device='CPU'; T in [DT_FLOAT]

	 [[{{node gradients/Mean_grad/Maximum}}]]
```

Temporary solution is always to cast:

```python
labels = tf.cast(labels, tf.float32)
```

But the real solution should switch on the flag that compiles all these ops with support for int32.

**Comment by phildow** [comment]
See https://github.com/tensorflow/tensorflow/issues/10192

Makefile checks:

```bash
# If ANDROID_TYPES is not set assume __ANDROID_TYPES_SLIM__
ifeq ($(ANDROID_TYPES),)
	ANDROID_TYPES := -D__ANDROID_TYPES_SLIM__
endif
```

Can override on command line?

```
--define ANDROID_TYPES=ANDROID_TYPES_FULL
````

Or define elsewhere with:

```
-DANDROID_TYPES=__ANDROID_TYPES_FULL__
```

**Comment by phildow** [comment]
See https://github.com/tensorflow/tensorflow/issues/4863

**Comment by phildow** [comment]
Issue has been addressed with the version 0.2.0 pod, but that pod won't lint because, as far as a i can tell, it vends a library larger than 2gb. tracking https://github.com/CocoaPods/CocoaPods/issues/8786
